demos/tests/run_tests.py

```diff
@@ -50,8 +53,26 @@ def parse_args():
         help='list of demos to run tests for (by default, every demo is tested)')
     parser.add_argument('--mo', type=Path, metavar='MO.PY',
         help='Model Optimizer entry point script')
+    parser.add_argument('--device_list', required=False, default="CPU, GPU",
+        help='List of devices to test')
+    parser.add_argument('--report_file', type=Path, required=False, default="demo_execution_time_report.csv",
```

Comment:
```suggestion
    parser.add_argument('--report-file', type=Path,
```

I think the default should be to not produce a report at all. Tools shouldn't dump stuff into the current directory without being asked to.